crowdnotifier-sdk/src/main/proto/qr.proto

```diff
@@ -1,16 +1,17 @@
-syntax = "proto2";
```

Comment:
We need to make sure that this and the file from libcrowdnotifier are the same...

Reply:
I just compared it to the latest libcrowdnotifier version and it matches.